.rspec

```diff
@@ -0,0 +1 @@
+--require spec_helper
```

Comment:
Why do we need this?

Reply:
The --require spec_helper line specifies that before running any tests, `spec_helper.rb` file should load. the `spec_helper.rb` contains configuration needed for all tests.

Reply:
In `rails_helper.rb` file, you are requiring this file. Do we need this?

Reply:
@SKH-WTAG bhai, file removed.